src/Compilers/CSharp/Portable/Syntax/InternalSyntax/CSharpSyntaxNode.cs

```diff
@@ -78,21 +78,8 @@ public override int RawContextualKind
             }
         }
 
-        public override bool IsStructuredTrivia
-        {
-            get
-            {
-                return this is StructuredTriviaSyntax;
-            }
-        }
-
-        public override bool IsDirective
-        {
-            get
-            {
-                return this is DirectiveTriviaSyntax;
-            }
-        }
+        public override bool IsStructuredTrivia => false;
```

Comment:
base methods appear to be virtual. Can/should these be deleted from here?

Reply:
you are correct.  deleted these.  thanks!